mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/jobcluster/proto/JobClusterManagerProto.java

```diff
@@ -1554,18 +1554,21 @@ public String toString() {
         }
     }
 
+    @ToString
     public static final class SubmitJobRequest extends BaseRequest {
         private final Optional<JobDefinition> jobDefinition;
         private final String submitter;
         private final String clusterName;
         private final boolean isAutoResubmit;
+        private final boolean submitLatest;
 
         @JsonCreator
         @JsonIgnoreProperties(ignoreUnknown = true)
         public SubmitJobRequest(
                 @JsonProperty("name") final String clusterName,
                 @JsonProperty("user") final String user,
-                @JsonProperty(value = "jobDefinition") final Optional<JobDefinition> jobDefinition) {
+                @JsonProperty(value = "jobDefinition") final Optional<JobDefinition> jobDefinition,
+                @JsonProperty("submitLatest") final boolean submitLatest) {
```

Comment:
If I understand correctly this flag is to submit from the current job cluster definition instead of the most recent job's definition (the latest job can have some overrides in its configs). maybe something like "submitLatestJobCluster".

Reply:
Yes, that's the intention. However, we should not allow overrides in the job definition config in the long run - this way, what's shown on the job cluster definition is what gets run from the job.